README.md

```diff
@@ -44,3 +44,39 @@ yarn run stylelint
 ```
 npm run stylelint
 ```
+
+## Prettier
+
+This config also includes a prettier config which can be extended to format `.scss`.
+
+Install [`prettier-stylelint`](https://github.com/hugomrdias/prettier-stylelint):
+
+```
+$ yarn add --dev prettier-stylelint
+```
+
+Extend the config in your `package.json`:
+
+```
+  "stylelint": {
+    "extends": [
+      "stylelint-config-shopify/prettier"
+    ]
+  },
+```
+
+Add a prettier config in `package.json`:
+
+```
```

Comment:
adding `json5` after the three backticks may improve readability of the code below